openfisca_us/variables/demographic/person.py

```diff
@@ -112,3 +112,55 @@ class provider_type_group(Variable):
     entity = Person
     label = u"CCDF provider type group"
     definition_period = YEAR
+
+
+class ccdf_hours_per_week(Variable):
+    value_type = float
+    entity = Person
+    label = u"Hours per week"
```

Comment:
```suggestion
    label = u"CCDF hours per week"
```
And elsewhere? Essentially just imagining us displaying a computation tree table somewhere in PolicyEngine, where the only things shown are the label and the values - probably guessable what it is in that context but this would make sure.